lib/interface_quda.cpp

```diff
@@ -5233,22 +5238,61 @@ void performWFlowQuda(QudaGaugeSmearParam *smear_param, QudaGaugeObservableParam
 
   gaugeObservables(in, obs_param[measurement_n]);
 
-  logQuda(QUDA_SUMMARIZE, "flow t, plaquette, E_tot, E_spatial, E_temporal, Q charge\n");
-  logQuda(QUDA_SUMMARIZE, "%le %.16e %+.16e %+.16e %+.16e %+.16e\n", smear_param->t0, obs_param[0].plaquette[0],
-          obs_param[0].energy[0], obs_param[0].energy[1], obs_param[0].energy[2], obs_param[0].qcharge);
+  QudaBoolean compute_plaq = obs_param[measurement_n].compute_plaquette;
```

Comment:
We're working towards using `bool` throughout QUDA.  Can you switch these to `auto` so that they automatically pick up this change?